### PR TITLE
commcare cloud install

### PR DESCRIPTION
**Pull request by snopoke**
I just had to setup my account on PNA and found some conflicting / differing instructions in `README.md` and `ansible/README.md`

There seems to be quite a bit of overlap between the two regarding setting up the environment. Any reason not to consolidate the install instructions into `README.md` and leave `ansible/REAMDE.md` for purely ansible info?

I was also thinking that it would be nice to have a script that you can just run from the web to do the full setup for you. Something like `bash <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)`.

It seems like `init.sh` is basically there. Curious to hear others thoughts.

other buddy @esoergel

### CODE REVIEW

File: control/init.sh [hunk 1]
```diff
@@ -39,17 +40,24 @@ wait
...
+    case $yn in
+        [Yy]* )
+            echo '[ -t 1 ] && source ~/init-ansible' >> ~/.profile
+        ;;
+        * )
+            echo 'You can always set it up later by running this command:'
+            echo "'[ -t 1 ] && source ~/init-ansible' >> ~/.profile"
+        ;;
+    esac
+fi
```
Comment (dannyroberts) [inline control/init.sh]: nice :)

**Comment by dannyroberts** [comment]
Yeah, anything that you see that doesn't look right between `README.md` and `ansible/README.md` probably isn't right and isn't intentional. I'm pro moving anything that seems not strictly ansible related out of `ansible/README.md` and into `README.md`.

**Comment by snopoke** [comment]
@dannyroberts think this is GTG now, I've tested it in a few varions of states.

Also update the output to use color.

File: README.md [hunk 2]
```diff
@@ -19,67 +19,21 @@ Docs at [https://dimagi.github.io/commcare-cloud/](https://dimagi.github.io/comm
...
-git pull
+sudo apt-get install git python-dev python-pip
+sudo pip install virtualenv virtualenvwrapper
 ```
 
-# Install commcare-cloud
-
-You must be in your virtualenv for the install script to work
+# Quick setup
+This should be run from your home director:
```
Comment (millerdev) [inline README.md]: nit: director -> directory

File: README.md [hunk 3]
```diff
@@ -19,67 +19,21 @@ Docs at [https://dimagi.github.io/commcare-cloud/](https://dimagi.github.io/comm
...
-
-at this time.
-
-**Note**: If you already have commcare-cloud cloned, then just enter that directory
-and update it with
+You will need python 2.7 and `virtualenvwrapper` installed to follow these instructions:
 
 ```
-git pull
+sudo apt-get install git python-dev python-pip
```
Comment (millerdev) [inline README.md]: This probably only works on debian-ish linux distros. Is it worth mentioning that?

**Comment by millerdev** [comment]
Travis failure:
```
/home/travis/.travis/job_stages: line 57: ./install.sh: No such file or directory

The command "./install.sh" failed and exited with 127 during .
```

File: README.md [hunk 4]
```diff
@@ -19,67 +19,21 @@ Docs at [https://dimagi.github.io/commcare-cloud/](https://dimagi.github.io/comm
...
+This should be run from your home directory:
 ```
-workon ansible
-```
-
-Then, simply run:
-
-```
-./install.sh
+source <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)
```
Comment (millerdev) [inline README.md]: @snopoke @dannyroberts looks like this needs some testing.
```
dmiller@linsrv:~$ virtualenv cchq-venv
New python executable in /home/dmiller/cchq-venv/bin/python
Installing setuptools, pip, wheel...done.
dmiller@linsrv:~$ . cchq-venv/bin/activate
(cchq-venv) dmiller@linsrv:~$ source <(curl -s https://raw.githubusercontent.com/dimagi/commcare-cloud/master/control/init.sh)
Could not open requirements file: [Errno 2] No such file or directory: '/dev/requirements.txt'
Downloading dependencies from galaxy and pip
The program 'ansible-galaxy' is currently not installed. To run 'ansible-galaxy' please ask your administrator to install the package 'ansible'
Requirement already up-to-date: pip in /home/dmiller/cchq-venv/lib/python2.7/site-packages
[1]-  Done                    pip install pip --upgrade
[2]+  Exit 127                ansible-galaxy install -r ${COMMCARE_CLOUD_REPO}/ansible/requirements.yml
-bash: ./control/check_install.sh: No such file or directory
-bash: /home/dmiller/.commcare-cloud/repo/control/.bash_completion: No such file or directory
Do you want to have the CommCare Cloud environment setup on login?

You can always set it up later by running this command:
'[ -t 1 ] && source ~/init-ansible' >> ~/.profile

Welcome to Ansible Control

Available commands:
update-code - update the ansible repositories (safely)
workon ansible - activate the ansible virtual env
ansible-deploy-control [environment] - deploy changes to users on this control machine
commcare-cloud - CLI wrapper for ansible.
                 See commcare-cloud -h for more details.
                 See commcare-cloud <env> <command> -h for command details.

Deprecated Commands
ap - Use commcare-cloud <env> ap instead.
aps - Use commcare-cloud <env> aps instead.
ae - Use commcare-cloud <env> run-shell-command instead.
(cchq-venv) dmiller@linsrv:/dev$
```
Note that it left me in `/dev` (odd directory to end up in). Did I do something wrong?

Comment (dannyroberts) [reply README.md]: @millerdev this was a result of https://github.com/dimagi/commcare-cloud/pull/1414 i think and not this PR. I have https://github.com/dimagi/commcare-cloud/pull/1421 open to fix.

**Pull request merged**